.eslintrc.js

```diff
@@ -124,7 +124,7 @@ module.exports = {
         // Require class properties and methods to explicitly use accessibility modifiers (public, private, protected)
         '@typescript-eslint/explicit-member-accessibility': 'error',
 
-        // Forbids an object literal to appear in a type assertion expression unless its used as a parameter. This
+        // Forbids an object literal to appear in a type assertion expression unless its used as a parameter (we violate this rule for test code, to allow for looser property matching for objects - more in the test-specific rules section below). This
```

Comment:
Super long comment. Should we break it up?

Reply:
Good call, fixed.